core/win32/events.mc

```diff
@@ -610,4 +610,11 @@ Application %1!s! (%2!s!). Cannot identify VFP frame offset.
 .
 ;#endif
 
+MessageId =
+Severity = Error
+Facility = DRCore
+SymbolicName = MSG_COULDNT_SYNCH_WITH_ALL_THREADS
+Language=English
+%1!s!:  DynamoRIO could not synch with all threads.
```

Comment:
Include the application name and pid, as all the other messages do: best to match them and start with that.

I would avoid "DynamoRIO": that's parameterized out for crash messages (xref dr_set_client_name(), PRODUCT_NAME, etc.), so best to avoid here.  Maybe just "Application %1!s! (%2!s!): Failed to synchronize with all threads when detaching."

Reply:
Done.